src/emu/softlist_dev.h

```diff
@@ -121,7 +121,7 @@ class software_list_device : public device_t
 	const std::list<software_info> &get_info() { if (!m_parsed) parse(); return m_infolist; }
 
 	// operations
-	const software_info *find(const std::string &look_for);
+	const software_info *find(std::string_view look_for);
```

Comment:
A string_view is bigger than a std::string &, what makes you choose one over the other?

Reply:
Because `const std::string &` forces the caller to have an `std::string`

I suppose my work on #9685 "inspired" this work; there was a place that I was forced to build an `std::string` in order to call `software_list_device::find`, which is of course lame.

Oh, and it is my understanding that @cuavas likes `std::string_view`